src/main/java/frc/robot/RobotContainer.java

```diff
@@ -4,37 +4,49 @@
 
 package frc.robot;
 
+import com.revrobotics.CANSparkMax;
+
+import edu.wpi.first.wpilibj.DoubleSolenoid;
 import edu.wpi.first.wpilibj.XboxController;
 import edu.wpi.first.wpilibj2.command.Command;
 import edu.wpi.first.wpilibj2.command.Commands;
+import edu.wpi.first.wpilibj2.command.button.Trigger;
 import frc.robot.Systems.*;
 import frc.robot.Systems.Chassis.*;
 import frc.robot.Commands.*;
+import frc.robot.Constants.Constants;
 
 public class RobotContainer {
-  private Chassis m_chassis;
+  //private Chassis m_chassis;
```

Comment:
Why comment out the drive code? I know it's not used for the shooter test, but shouldn't interfere with it